README.md

```diff
@@ -0,0 +1,11 @@
+## Development
+
+### Installation
+
+1. Clone this repository `git clone git@github.com:bigtestjs/bigtest.git`
```

Comment:
Why SSH and not HTTPS?

Reply:
No special reason, other than that is my default. In my experience, SSH is better because it doesn't require password to be entered as when using HTTPS (although, I'm not sure if it's possible to accomplish passwordless authentication with HTTPS)

Reply:
I use HTTPS because GitHub [recommends it for a number of reasons](https://help.github.com/en/github/using-git/which-remote-url-should-i-use#cloning-with-https-urls-recommended) and the credentials are [easily saved](https://help.github.com/en/github/using-git/caching-your-github-password-in-git).